agency/python/src/test_reputation.py

```diff
@@ -473,11 +474,34 @@ def test_precision(self):
 
 
 class TestReputationServiceAdvanced(TestReputationServiceParametersBase):
-
+	def clear(self):
```

Comment:
@nejc9921 - are you sure you need to duplicate the clear function in subclass? If not so, please remove the redundant code and repeat the PR, please